### ISSUE TITLE
parallelTesting option

### ISSUE THREAD

**Issue by oallauddin** [damianszczepanik/maven-cucumber-reporting#167]
What does the parallelTesting option control? I could not find it in the documentation.

**Comment by sanketchandak** [comment]
Any update on this?

**Comment by damianszczepanik** [comment]
Where does it declared? Can't see in https://github.com/damianszczepanik/maven-cucumber-reporting/blob/master/src/main/java/net/masterthought/cucumber/CucumberReportGeneratorMojo.java

**Comment by gogonutsss** [comment]
It is mentioned on the READme pom.xml example when you land in the project. This attribute also errors out in intellij (not in eclipse).
![Selection_026](https://user-images.githubusercontent.com/329338/103620091-8fd99c00-4f3b-11eb-9554-7bd94a1d37ee.png)

**Comment by JitendraSingh2TR** [comment]
I guess it is removed. This option was present in previous versions.

**Comment by itbsg** [comment]
yes, it has been removed in [cucumber-reporting](https://github.com/damianszczepanik/cucumber-reporting), see https://github.com/damianszczepanik/cucumber-reporting/commit/078798351ed9716cfeeda70536302f53bd1e93ab#diff-6d582fceb08c5692ce8f3116f860bd3c75fc9a121d5df55a871e30d9976a3d8b

Thus, README.md should be updated. I'll create a small MR for that this week.
